### PR TITLE
chore: various metrics tasks

### PR DESCRIPTION
**Pull request by hikchoi**
# chore: various metrics tasks
This PR contains various tasks related to metrics.
- [x] CLI telemetry is now disabled when run by CI
- [x] Inactive users (users who were active on week 1, but has been inactive for at least 4 weeks) will be prompted for a survey similar to the lapsed user survey (with slightly changed wordings). They will be re-prompted if they reject and another 4 week has passed.
- [x] Remove tracking of `LookupModifierSetByController`
- [x] Add tracking of `LookupModifierSetOnAccept`

# Segment verification
- [x] CLI telemetry events are not sent when in CI (`SCRIPT_BUILD_ENV` set to `ci`)
- [x] Inactive users survey sends correct events
- [x] `LookupModifierSetByController` event is not sent
- [x] `LookupModifierSetOnAccept` events are correctly sent on lookup accept.

# Documentation
- [x] update user survey section
- [x] update on removed events and added events for lookup modifiers

# Pull Request Checklist

You can go to [dendron pull requests](https://wiki.dendron.so/notes/adc39825-77a6-46cf-9c49-2642fcb4248e.html) to see full details for items in this checklist.

### Quality Assurance
- [x] Add a [test](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#writing-tests) for the new feature
- [x] Make sure all the existing [tests](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#running-all-tests) pass
- [x] Do a spot check by running your feature with our [test workspace](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#test-workspace)
- [~] If your tests involve running [[manual Testing|dendron://dendron.dendron-site/dendron.dev.qa.test#manual-testing]], make sure to update [[Test Workspace|dendron://dendron.dendron-site/dendron.dev.ref.test-workspace]] with any necessary notes/additions to perform manual test ^9jtWc6ov340p
- [x] After you submit your pull request, check the output of our [integration test](https://github.com/dendronhq/dendron/actions) and make sure all tests pass
  - NOTE: If you running MacOS or Linux, pay special attention to the Windows output and vice versa if you are developing on Windows

#### Special Cases
- [~] If your tests changes an existing snapshot, make sure that snapshots are [updated](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#updating-test-snapshots)
- [~] If you are adding a new language feature (graphically visible in VS Code/preview/publishing), make sure that it is included in [test-workspace](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#test-workspace). We use this to manually inspect new changes and for auto regression testing 

### Docs
- [x] Make sure that the PR title follows our [commit style](https://wiki.dendron.so/notes/adc39825-77a6-46cf-9c49-2642fcb4248e.html#commit-style)
- [x] Please summarize the feature or impact in 1-2 lines in the PR description
- [x] If your change reflects documentation changes, also submit a PR to [dendron-site](https://github.com/dendronhq/dendron-site) and mention the doc PR link in your current PR

### Git
- [x] Make sure your branch names adhere to our commit style [[#^6zdCscSXs1MM]]
    - All PRs should start with `[feat|fix|enhance|]/[{description-of-pr-in-kebab-case}]`
        - `eg. feat/add-thisthing`

### Analytics
- [x] If you are adding analytics related changes, make sure the [Telemetry](https://wiki.dendron.so/notes/84df871b-9442-42fd-b4c3-0024e35b5f3c.html) docs are updated

### Github Issue
- [~] If this pull request is addressing an existing issue, make sure to link this PR to the issue that it is resolving.
- [~] If the resolution comes with a document update, link the docs PR to the issue as well.

### CODE REVIEW

**Comment by hikchoi** [comment]
DOCS: https://github.com/dendronhq/dendron-site/pull/293

**Comment by kevinslin** [comment]
Looks good. Can we document the survey flow in a note(s)?
* area.growth.ref.survey.lapsed-user
* area.growth.ref.survey.inactive-user
* area.growth.ref.survey.new-user

**Review by kevinslin — CHANGES_REQUESTED** [review]

File: packages/dendron-cli/src/utils/analytics.ts [hunk 1]
```diff
@@ -3,19 +3,23 @@ import { SegmentUtils } from "@dendronhq/common-server";
...
-    SegmentUtils.track(event, { type: "cli", cliVersion }, props);
+    if (!process.env.GITHUB_ACTIONS) {
+      SegmentUtils.track(event, { type: "cli", cliVersion }, props);
+    }
   }
 
   static async trackSync(event: string, props?: any) {
     const cliVersion = process.env.npm_package_version!;
-    await SegmentUtils.trackSync(event, { type: "cli", cliVersion }, props);
+    if (!process.env.GITHUB_ACTIONS) {
```
[the remark this answers is not in the document]
Comment (hikchoi) [reply packages/dendron-cli/src/utils/analytics.ts:13]: added tests

File: packages/plugin-core/src/commands/NoteLookupCommand.ts [hunk 2]
```diff
@@ -376,6 +376,13 @@ export class NoteLookupCommand extends BaseCommand<
       duration: profile,
       isNew,
     });
+    const metaData = MetadataService.instance().getMeta();
+    if (_.isUndefined(metaData.firstLookupTime)) {
+      MetadataService.instance().setFirstLookupTime();
+      MetadataService.instance().setLastLookupTime();
```
[the remark this answers is not in the document]
Comment (hikchoi) [reply packages/plugin-core/src/commands/NoteLookupCommand.ts:382]: fixed

**Review by kevinslin — APPROVED** [review]

**Pull request merged**